internal/bundle/helpers.go

```diff
@@ -56,21 +57,21 @@ func writeConfigFile(t *testing.T, config map[string]any) (string, error) {
 }
 
 func validateBundle(t *testing.T, ctx context.Context, path string) ([]byte, error) {
-	t.Setenv("BUNDLE_ROOT", path)
+	ctx = env.Set(ctx, "BUNDLE_ROOT", path)
```

Comment:
`t.Setenv` prevents tests from running in parallel (the env is global state).

We use `libs/env` everywhere to access environment variables. This allows for overriding actual environment variables in the context. This in turn allows us to run tests in parallel.